src/App.tsx

```diff
@@ -1,26 +1,131 @@
-/* eslint-disable max-len */
+/* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useMemo, useState } from 'react';
 import { UserWarning } from './UserWarning';
-
-const USER_ID = 0;
+import { Todo } from './types/Todo';
+import { TodoList } from './components/TodoList/TodoList';
+import { FilterTypes } from './types/FilterTypes';
+import { getPreparedTodos } from './utils/GetPreparedTodos';
+import { getIdsOfActiveTodos } from './utils/getAmountOfActiveTodos';
+import { Footer } from './components/Footer/Footer';
+import { Errors } from './utils/Errors';
+import { ErrorsField } from './components/ErrorsField/ErrorsField';
+import { Header } from './components/Header/Header';
+import { USER_ID } from './utils/CONSTANTS';
+import { deletingTodo } from './utils/deletingTodo';
+import { handleGetTodos } from './utils/handleGetTodos';
+import { updatingTodo } from './utils/updatingTodo';
+import { TodoOmited } from './types/TodoOmited';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [filterInstructions, setFilterInstructions] = useState<FilterTypes>(
+    FilterTypes.All,
+  );
```

Comment:
will be better and more readable
```suggestion
  const [
    filterInstructions,
    setFilterInstructions
  ] = useState<FilterTypes>(FilterTypes.All);
```